android/java/org/chromium/chrome/browser/crypto_wallet/activities/BraveWalletActivity.java

```diff
@@ -243,15 +244,15 @@ private void setNavigationFragments(int type) {
             setupWalletFragment.setOnNextPageListener(this);
             navigationItems.add(new NavigationItem(
                     getResources().getString(R.string.setup_crypto), setupWalletFragment));
-            mBraveWalletService.onOnboardingShown();
+            mBraveWalletP3A.reportOnboardingAction(OnboardingAction.SHOWN);
         } else if (type == UNLOCK_WALLET_ACTION) {
             UnlockWalletFragment unlockWalletFragment = new UnlockWalletFragment();
             unlockWalletFragment.setOnNextPageListener(this);
             navigationItems.add(new NavigationItem(
                     getResources().getString(R.string.unlock_wallet_title), unlockWalletFragment));
         } else if (type == RESTORE_WALLET_ACTION) {
             mShowBiometricPrompt = false;
-            RestoreWalletFragment restoreWalletFragment = new RestoreWalletFragment();
+            RestoreWalletFragment restoreWalletFragment = new RestoreWalletFragment(false);
```

Comment:
we should not pass arguments to constructor. instead we should pass the argument to fragment.

Reply:
can you be more specific